platformio_esp32_solo1.ini

```diff
@@ -18,7 +18,7 @@ build_unflags             = ${esp32_base.build_unflags}
 ; IDF 5.1.2
 [esp32_solo1_common_LittleFS]
 extends                   = esp32_base_idf5
-platform_packages         = framework-arduino-solo1 @ https://github.com/Jason2866/esp32-arduino-lib-builder/releases/download/1895/framework-arduinoespressif32-solo1-release_v5.1-8f01299.zip
+platform_packages         = framework-arduino-solo1 @ https://github.com/Jason2866/esp32-arduino-lib-builder/releases/download/1915/framework-arduinoespressif32-solo1-release_v5.1-d4d5f8a.zip
```

Comment:
Is there a difference with /1916/ in the `core_defs` file?

Reply:
Not sure what you mean?